tool/cli.ts

```diff
@@ -77,32 +77,28 @@ program
     tryOrExit(({ args, options, logger }) => {
       const { locales } = args;
       const { strict } = options;
-      let fine = true;
       if (strict) {
         for (const locale of locales) {
           try {
             Redirect.validateLocale(locale, strict);
             logger.info(chalk.green(`✓ redirects for ${locale} looking good!`));
           } catch (e) {
-            logger.info(
+            logger.error(
               chalk.red(`_redirects.txt for ${locale} is causing issues: ${e}`)
             );
-            fine = false;
+            throw new Error("🔥 Errors loading redirects 🔥");
```

Comment:
If we stick with throwing, can't we throw directly, rather than logging the error first?

```suggestion
            throw new Error(`_redirects.txt for ${locale} is causing issues: ${e}`);
```

Reply:
Good point! I couldn't apply the suggestions directly here 
> GitHub: Applying suggestions on deleted lines is currently not supported.